docs/docs/target/kubernetes.md

```diff
@@ -101,13 +91,79 @@ Total: 3 (UNKNOWN: 0, LOW: 1, MEDIUM: 0, HIGH: 2, CRITICAL: 0)
 └────────────────┴────────────────┴──────────┴────────┴───────────────────┴──────────────────────────────────┴───────────────────────────────────────────────────┘
 ```
 
+### Taints and Tolerations
+
+The node-collector scan-job will run on every node. In case the node has been tainted, it is possible to add toleration to the scan job for it to be scheduled on the tainted node
```

Comment:
Could we add the link to the tainted nodes? :D https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/ just to make sure people have something to reference if they don't know what it means?